google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BuiltinMetricsTracer.java

```diff
@@ -85,6 +87,8 @@ class BuiltinMetricsTracer extends BigtableTracer {
 
   private Long serverLatencies = null;
 
+  private String target_endpoint = "unspecified";
```

Comment:
does this need to be volatile? please check that the thread that sets the target is the same as the thread that reads it